### PR TITLE
Reduce number of builds.

### PR DESCRIPTION
**Pull request by amouat**
Signed-off-by: Adrian Mouat <adrian@chainguard.dev>

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -38,6 +38,7 @@ jobs:
         config: apko.yaml
         image_refs: apko.images
         base-tag: ghcr.io/${{ github.repository }}
+        archs: x86_64,aarch64,armv7
```
Comment (imjasonh) [inline .github/workflows/release.yaml:41]: I know at least one downstream project that depends on building s390x images on this base. Can we do everything but maybe riscv64le and 386?

Comment (amouat) [reply .github/workflows/release.yaml:41]: Sure. I was actually a bit torn about this as if there's one image that should build for all archs, it's this one.

Comment (imjasonh) [reply .github/workflows/release.yaml:41]: I agree. Can we have an issue to re-enable the archs we're disabling here? I'd like to get back to having them all.

**Comment by kaniini** [comment]
I would rather not do this.

**Comment by imjasonh** [comment]
For visibility, AIUI we're disabling 386 because of a flake in https://github.com/distroless/static/runs/7461126293?check_suite_focus=true and because we don't believe there's anybody relying on this image being built for 386.

The GCT distroless image previously used by Tekton supports:
- linux/amd64
- linux/arm64
- linux/arm (no variant specified)
- linux/ppc64le
- linux/s390x

So we should aim to provide the same at least, unless there's some reason we absolutely can't.

**Comment by amouat** [comment]
I think it was just a flake, probably unrelated to 386. 

Let's not change release.yaml at all.

I was considering cutting the pre-submit builds to just one, but static should be so quick it seems pointless. Just closing this for the moment.

**Pull request closed without merging**